src/devices/bus/coco/coco_gmc.h

```diff
@@ -0,0 +1,32 @@
+// license:BSD-3-Clause
+// copyright-holders:tim lindner
+#pragma once
+
+#ifndef __COCO_GMC_H__
+#define __COCO_GMC_H__
```

Comment:
Symbols starting with underscores are bad; make this MAME_DEVICES_BUS_COCO_COCO_GMC_H